optuna/storages/rdb/storage.py

```diff
@@ -575,6 +572,9 @@ def _update_trial(
                 New user_attr. None when there are no updates.
             system_attrs:
                 New system_attr. None when there are no updates.
+            datetime_complete:
+                Completion time of the trial. Set only if and only if this method
```

Comment:
I'm not sure but it seems there is an extra "only".
```suggestion
                Completion time of the trial. Set if and only if this method
```

Reply:
Yes, it is. I fixed the typo. Thank you.